### PR TITLE
Persist breadcrumb state between sessions

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/riot-web/issues/8549

### CODE REVIEW

File: src/components/views/rooms/RoomBreadcrumbs.js [hunk 1]
```diff
@@ -35,21 +35,42 @@ export default class RoomBreadcrumbs extends React.Component {
...
+            }
+        }
     }
 
     componentWillUnmount() {
         dis.unregister(this._dispatcherRef);
     }
 
-    componentDidUpdate() {
+    componentDidUpdate(props, state) {
```
Comment (bwindels) [inline src/components/views/rooms/RoomBreadcrumbs.js]: nit: doesn't look like you're using these?

Comment (turt2live) [reply src/components/views/rooms/RoomBreadcrumbs.js]: I'm using state, but the props get in the way of the ordering.

Comment (bwindels) [reply src/components/views/rooms/RoomBreadcrumbs.js]: I can see you are using `this.state`, but fail to spot usage of `state`?

Comment (turt2live) [reply src/components/views/rooms/RoomBreadcrumbs.js]: That would be a bug that I failed to catch ._.

Comment (turt2live) [reply src/components/views/rooms/RoomBreadcrumbs.js]: ... and by that I mean I'll take out the `prop, state` declarations because they aren't doing much

**Pull request merged**